rmgpy/rmg/mainTest.py

```diff
@@ -116,6 +117,19 @@ def testRMGExecute(self):
         self.assertTrue(len(self.rmg.reactionModel.edge.reactions) > 0)
         self.assertTrue(len(self.rmg.reactionModel.edge.species) > 0)
         
+        #test seed mech generation
+        seedDir = os.path.join(originalPath,self.dir_name,'seed')
+        self.assertTrue(os.path.exists)
+        os.chdir(seedDir)
+        self.assertTrue(os.path.exists(self.rmg.name+'.py')) #thermo library made
+        self.assertTrue(os.path.exists(self.rmg.name)) #kinetics library folder made
+        kDir = os.path.join(seedDir,self.rmg.name)
+        os.chdir(kDir)
+        self.assertTrue(os.path.exists('dictionary.txt')) #dictionary file made
+        self.assertTrue(os.path.exists('reactions.py')) #reactions file made
+        
+        os.chdir(os.path.join(originalPath,self.dir_name)) #return to original directory
+        
```

Comment:
Now we load and check that we've properly loaded the seed mech through both the library and seed mechanism functions.